cpp/test/Ice/custom/AllTests.cpp

```diff
@@ -54,7 +54,7 @@ allTests(Test::TestHelper* helper)
 
     Test::TestIntfPrxPtr t = Ice::checkedCast<Test::TestIntfPrx>(base);
     test(t);
-    test(Ice::targetEqualTo(t, base));
+    test(t == base);
```

Comment:
We test over and over over checkedCast; while I kept it as-is in this PR, I think we should get rid of all this code, and only test checkedCast in one test.